express/services/players.js

```diff
@@ -82,11 +85,26 @@ function remove(id) {
     });
 }
 
+function getBestPlayers() {
+    const rawQuery = `SELECT DISTINCT ON ("team"."teamName") "teamName",
```

Comment:
This query can be moved to the const in the top of the file. 
Please style it properly, indents are confusing.

Reply:
Done